lnst/RecipeCommon/Perf/Evaluators/BaselineEvaluator.py

```diff
@@ -47,14 +47,23 @@ def evaluate_group_results(
         result_text = self.describe_group_results(recipe, recipe_conf, results)
 
         baselines = self.get_baselines(recipe, recipe_conf, results)
-        for result, baseline in zip(results, baselines):
+        result_index = len(recipe.current_run.results)
+
+        for i, (result, baseline) in enumerate(zip(results, baselines)):
             comparison, text = self.compare_result_with_baseline(
-                recipe, recipe_conf, result, baseline
+                recipe, recipe_conf, result, baseline, result_index
             )
             comparison_result = ResultType.max_severity(comparison_result, comparison)
             result_text.extend(text)
 
-        recipe.add_result(comparison_result, "\n".join(result_text))
+        recipe.add_result(
+            comparison_result,
+            "\n".join(
+                [
+                    str(text) for text in result_text  # result text contains thresholds in dict form
```

Comment:
i don't think this comment is correct? looking at the flow or cpu baseline evaluators, both of them return simple strings right?

Reply:
conversion to string is just leftover, as i moved thresholds prints it's not necessary anymore, thanks.